<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColimateZeroAjustDialog</class>
 <widget class="QDialog" name="ColimateZeroAjustDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>准直器零点标定</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>射线源钨门水平</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>射线源钨门垂直</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>面阵钨门水平</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>面阵钨门垂直</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>线阵探测器层厚</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="rayHEdit"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="rayVEdit"/>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="detHEdit"/>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="detVEdit"/>
   </item>
   <item row="1" column="4">
    <widget class="QLineEdit" name="layerThicknessEdit"/>
   </item>
   <item row="2" column="0" colspan="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>356</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
